uahf-technical-spec.md

```diff
@@ -199,6 +199,13 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-7 Difficulty adjustement in case of hashrate drop
+
+In case the MTP of the tip of the chain is 12h after the MTP 6 block before,
+the proof of work target is increased by 1/4, which correspond to a reduction
+of the difficulty of 20% .
```

Comment:
better to write:

"to a difficulty reduction of 20%"

get rid of some needless verbosity.